java/daikon/inv/unary/OneOf.java.jpp

```diff
@@ -2118,8 +2118,9 @@ public String format_csharp_contract(@GuardSatisfied ONEOFSCALAR this) {
     if (num_elts == 0) {
       return false;
     }
-
-    if (!(state instanceof INT[])) {
+    @SuppressWarnings("allcheckers:operand.instanceof.subtype")
+    boolean stateTypeMatch = !(state instanceof INT[]); //  Warnings are suppressed because only the data type of the state needs to be checked and not the annotations
```

Comment:
Please place the comment on the line with `@SuppressWarnings`, because the comment is about the warning suppression.

Reply:
I have changed the comment's location in my latest commit.